plugins/catalog/src/js/repositories/RepositoriesList.js

```diff
@@ -8,16 +8,11 @@ import "rxjs/add/operator/switchMap";
 import { componentFromStream, graphqlObservable } from "data-service";
 
 // tools to make easier to create schemas and queries
-import { makeExecutableSchema } from "graphql-tools";
+// import { makeExecutableSchema } from "graphql-tools";
```

Comment:
Do we still want to keep the comment?

Reply:
I would say yes, it is commented because in this case we got the schema built on line 18. But I think this comment goes well with line 12 where we explain how it is built. But if I move them closer, when you uncomment the linter yells at you because of the import order. 

It is not ideal, but I would leave it like this.